crates/starknet-devnet-server/src/api/json_rpc/error.rs

```diff
@@ -58,6 +58,8 @@ pub enum ApiError {
     NoStateAtBlock { msg: String },
     #[error(transparent)]
     HttpApiError(#[from] HttpApiError),
+    #[error("the compiled class hash did not match the one supplied in the transaction")]
```

Comment:
I don't like the fact that we define these errors in two places (this is not the first such error, but might be the first one where I used exactly the same string in both definitions - copied it from the rpc specs).

Reply:
@marioiordanov have you already faced this situation? Did I do it the expected way?

Reply:
I dont see a major problem with it.